mcrouter/scripts/install_ubuntu_14.04.sh

```diff
@@ -4,6 +4,8 @@ set -ex
 
 [ -n "$1" ] || ( echo "Install dir missing"; exit 1 )
 
+
+sudo apt-get install software-properties-common
```

Comment:
Is this change incidental? If it fixes the current Dockerfile, then maybe a separate PR would be good while this one is being reviewed.